src/Indice.Features.Cases.AspNetCore/Workflows/Activities/SendMessageActivity.cs

```diff
@@ -16,7 +16,7 @@ namespace Indice.Features.Cases.Workflows.Activities
     [Activity(
         Category = "Cases",
         DisplayName = "Send Message",
-        Description = "Send a checkpoint change and/or a comment and/or an attachment to a case. The current context user will be used.",
+        Description = "Send a checkpoint change and/or a comment and/or an attachment and/or data update to a case. The current context user will be used.",
```

Comment:
Send a message to a case in order to change the active checkpoint, update its data, add an attachment or add a comment. The current context user will be responsible for the change.